### ISSUE TITLE
Webpack HMR in Production

### ISSUE THREAD

**Issue by christophwolff** [nuxt/nuxt.js#1669]
Is it correct that Webpack HMR is running in production? I tought it is only for development.

**Comment by christophwolff** [comment]
Maybe a little more context here. I created a little plugin that gives me the current path everytime a user visits a route:

```
export default ({ app: { router, store } }) => {
    router.afterEach((to, from) => {
    console.log(to.path)
  })
}
```

When i start the server with npm run start every second this happens.

<img width="962" alt="nuxthmt" src="https://user-images.githubusercontent.com/2046613/30516323-f9f3139e-9b3b-11e7-9082-168e92917119.png">

You can see that i visited that `/podcast` route by hand. 

So someone knows what this is about? I think for big sites that relate on analytics this is a serious because the the analytics is broken. I mean you can filter this in that Script but iam asking myself why this happens. Anyone? 
@Atinux ?

**Comment by Atinux** [comment]
@christophwolff Are you sure that your `npm start` script run `nuxt start`?

**Comment by christophwolff** [comment]
@Atinux It is really strange. now the error seems gone and everything just works. Yesterday i played around with a middleware and this seems to work back then. Now i wanted to recreate the error with a plugin and its gone. Sorry.  :-/

**Comment by Atinux** [comment]
It's possible that you left an open tab when you were working with nuxt dev, and it was trying to reconnect the socket

**Comment by vaggelis2018** [comment]
@Atinux i have the same problem. This makes localization on error page not possible because it always gets only one param and it is __webpack_hmr

**Comment by christophwolff** [comment]
For me is like what @Atinux said. I left one tab open and oversaw it.

**Comment by vaggelis2018** [comment]
@christophwolff already checked it and it happens only in error layout.

**Comment by elcobvg** [comment]
Has this issue already been resolved? I'm experiencing the exact same problem: as soon as I run nuxt start, the app calls /__webpack_hmr No tab left open or anything like that.

**Comment by christophwolff** [comment]
This happens for me, when i use it with Safari on iOS or Mac OS. And it contantly reloads. I think this will be solved in Nuxt 2.0

**Comment by besnikh** [comment]
@Atinux my body was shaking after seeing webpack in production.. then I saw your comment :) :+1:
